bin/docker/modify.sh

```diff
@@ -27,15 +25,4 @@ if [ "${COUNTLY_CONTAINER}" != "frontend" ]; then
 		python3.8 -m pip install -r /opt/countly/plugins/ab-testing/api/bayesian/requirements.txt
 		cd /opt/countly/plugins/ab-testing/api/bayesian && python3.8 model.py
 	fi
-
-	# install monngodb tools to have mongoexport
-	if [ "${ID}" == "debian" ] || [ "${ID}" == "ubuntu" ]; then
-	    wget https://fastdl.mongodb.org/tools/db/mongodb-database-tools-debian10-x86_64-100.5.4.deb
-            apt install -y --allow-downgrades ./mongodb-database-tools-*-100.5.4.deb
-	    rm ./mongodb-database-tools-*-100.5.4.deb
-	else
-            wget https://fastdl.mongodb.org/tools/db/mongodb-database-tools-rhel70-x86_64-100.5.4.rpm
-            rpm -i  ./mongodb-database-tools-*-100.5.4.rpm
-	    rm ./mongodb-database-tools-*-100.5.4.rpm
-	fi
```

Comment:
> @ar2rsawseen is the user export still supported via the API? I can remember that this was added because the export was based on the mongoexport binaries present.

So mongoexport was removed from the image because it is not a dependency anymore because user export now is done purely through nodejs driver, it basically exports data into a new collection and then on download streams that collection, and on delete - deletes that collection.
As far as I remember all the API and interface where left the same, and only export file itself was different